src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/AppModel/ContentFilePart.cs

```diff
@@ -103,15 +102,15 @@ protected override string GetContentTypeCore()
         #region Private Methods
 
 
-        private Uri GetEntryAssemblyLocation()
+        private string GetEntryAssemblyLocation()
         {
-            Uri entryLocation = null;
+            string entryLocation = null;
 
             try
             {
-                #pragma warning disable // Type or member is obsolete
-                entryLocation = new Uri(Application.ResourceAssembly.CodeBase);
-                #pragma warning restore // Type or member is obsolete
+                // We do not want to use Application.ResourceAssembly.CodeBase
```

Comment:
This comment seems weird to me. In my opinion, we shouldn't need to justify in a comment the non-usage of an obsoleted API.

Reply:
@ThomasGoulet73 Thank you and I fix it.